### PR TITLE
Added option to use proxy for curl command

### PR DESCRIPTION
**Pull request by dutsmiller**

### CODE REVIEW

**Comment by craigwatson** [comment]
Thanks for the PR - nice addition!

Can you modify the ERB code to use `scope.lookupvar` as per the other variables in the file?

**Comment by dutsmiller** [comment]
Hope that's what you were looking for, I'm a novice at public PRs :)

File: templates/download.sh.erb [hunk 1]
```diff
@@ -13,12 +13,14 @@ function check_exec {
 DOWNLOAD_URL="<%= scope.lookupvar('vmwaretools::archive_url') %>/VMwareTools-<%= scope.lookupvar('vmwaretools::version') %>.tar.gz"
 DOWNLOAD_LOCATION="<%= scope.lookupvar('vmwaretools::working_dir') %>/VMwareTools-<%= scope.lookupvar('vmwaretools::version') %>.tar.gz"
 GOOD_CHECKSUM="<%= scope.lookupvar('vmwaretools::archive_md5') %>"
+CURL_PROXY="<%= scope.lookupvar('vmwaretools::curl_proxy') %>"
```
Comment (craigwatson) [inline templates/download.sh.erb]: You shouldn't need to declare this variable, you can just change L22 to use `scope.lookupvar` for the `-x` parameter.

**Comment by craigwatson** [comment]
No problem - don't worry about the Travis CI failures, I'm aware of an issue with my module and the newer Travis infrastructure.

All looks good apart from the comment above  :)

**Comment by dutsmiller** [comment]
works for me, I like simple

**Pull request merged**